src/enclave_proc/resource_manager.rs

```diff
@@ -615,11 +630,24 @@ impl EnclaveHandle {
             .add_info(vec!["memory", &requested_mem_mib.to_string()]));
         }
 
+        let link_path = read_link(PathBuf::from(format!(
+            "/proc/self/fd/{}",
+            self.eif_file.as_ref().unwrap().as_raw_fd()
+        )));
+        let eif_path = match link_path {
+            Ok(link_path) => String::from(
+                link_path
+                    .to_str()
+                    .unwrap_or("Invalid unicode EIF file name"),
+            ),
+            Err(_) => String::from("Unknown file"),
+        };
         let eif_file = self.eif_file.as_mut().ok_or_else(|| {
             new_nitro_cli_failure!(
                 "Failed to get mutable reference to EIF file",
                 NitroCliErrorEnum::FileOperationFailure
             )
+            .add_info(vec![eif_path.as_str(), "Get EIF reference"])
```

Comment:
This doesn't fail because of the EIF itself, I don't think, so reporting the file path is not needed here.

Reply:
Removed it in the latest revision.